src/Workspaces/Remote/ServiceHub/Services/Renamer/RemoteRenamerService.cs

```diff
@@ -79,12 +79,14 @@ private CodeCleanupOptionsProvider GetClientOptionsProvider(RemoteServiceCallbac
                 if (symbol == null)
                     return null;
 
-                var fallbackOptions = GetClientOptionsProvider(callbackId);
-
-                var result = await LightweightRenameLocations.FindRenameLocationsAsync(
-                    symbol, solution, options, fallbackOptions, cancellationToken).ConfigureAwait(false);
-
-                return result.Dehydrate();
+                var renameLocations = await SymbolicRenameLocations.FindLocationsInCurrentProcessAsync(
+                    symbol, solution, options, GetClientOptionsProvider(callbackId), cancellationToken).ConfigureAwait(false);
```

Comment:
there was no point jumping to LightweightRenameLocations on the *oop* side.  it exists on the host side as a temporary data so we can call back into oop efficiently without rehydrating symbols.  THe previous code would make the LightweightRenameLocations  then immediately convert that to SymbolicRenameLocations.  so this just bypasses that.

Reply:
Is it possible we can annotate types that shouldn't be used in OOP/Not OOP locations based on folder paths? I assume outside of serialization everything in ServiceHub/* doesn't need to use serializable data.

Reply:
Definitely an good idea! :)